### PR TITLE
Old atlassian pat

### PR DESCRIPTION
**Pull request by babenek**
## Description

- Add Atlassian old PAT pattern

## How has this been tested?

Please describe the tests that you ran to verify your changes.

- [x] UnitTest
- [x] Benchmark

### CODE REVIEW

File: credsweeper/rules/config.yaml [hunk 1]
```diff
@@ -724,6 +724,17 @@
     - src
     - doc
 
+- name: Atlassian Old PAT token
+  severity: info
+  type: pattern
+  values:
+    - (^|[^0-9A-Za-z.$_/+-])(?P<value>[a-zA-Z0-9]{24})([^0-9A-Za-z.$_/+-]|$)
```
Comment (babenek) [inline credsweeper/rules/config.yaml:731]:
```suggestion
    - (^|[^0-9A-Za-z.$_/+-])(?P<value>[a-zA-Z0-9]{24})([^=0-9A-Za-z.$_/+-]|$)
```

File: credsweeper/common/morpheme_checklist.txt [hunk 2]
```diff
@@ -584,7 +582,7 @@ fire
 firm
 first
 fix
-flash
+flas
 flat
```
Comment (csh519) [inline credsweeper/common/morpheme_checklist.txt:586]: Could you explain why the keyword list modified?
Is there any data before and after?

Comment (babenek) [reply credsweeper/common/morpheme_checklist.txt:586]: 'ash' was added and it conflicts with 'flash', 'hash' ...

Comment (csh519) [reply credsweeper/common/morpheme_checklist.txt:586]: Okay, then why the `ash` is added?
I'm not talking about the `flash` case only, but all.

Comment (babenek) [reply credsweeper/common/morpheme_checklist.txt:586]: I don't remember exactly.
- it helps filter out 'hash_xxxxx...' creds - exist 'has' and 'ash'
- 'ash' is an independent word.

**Review by iuriimet — APPROVED** [review]

**Pull request merged**